app/src/main/res/values/strings.xml

```diff
@@ -428,4 +428,6 @@
   <string name="coming_soon">Coming Soon</string>
   <string name="recommended_stories">Recommended Stories</string>
   <string name="stories_for_you">Stories For You</string>
+  <string name="on">ON</string>
+  <string name="off">OFF</string>
```

Comment:
We should make these names more specific.

Also, why the capitlization?

Reply:
Rename to `switch_on_content_description` ?

I have done capitalisation because I have seen this same format getting used in `Switches` which are in `Settings`.
Example:
<img src="https://user-images.githubusercontent.com/9396084/108902055-a0f08080-7641-11eb-9893-79be6558d66c.png" width="250" />